card/mod/standard/set/abstract/wql_search.rb

```diff
@@ -22,9 +22,8 @@ def fetch_query args={}
 end
 
 def standardized_query_args args
-  args.symbolize_keys!
   args[:context] ||= cardname
-  args
+  args.symbolize_keys
```

Comment:
oh, probably not.  but the old code had a problem in certain contexts.  the exclamation point was breaking things when args that came in were a Card::Controller params object.  (I guess it doesn't have that method because it doesn't let you alter the keys on the object).